src/components/PermissionsManagement/UserPermissionsPopUp.jsx

```diff
@@ -87,9 +81,14 @@ const UserPermissionsPopUp = ({ allUserProfiles, toggle, getAllUsers }) => {
       })
       .catch(err => console.log(err));
     getAllUsers();
-    console.log(newUserInfo, '2');
 
-    toast.success('Permission has been updated successfully');
+    const SUCCESS_MESSAGE = `
```

Comment:
Looks good. I like that you are making sure that everyone knows about the updated permissions.